command/debug.go

```diff
@@ -731,35 +731,35 @@ func (c *DebugCommand) collectPprof(ctx context.Context) {
 
 		var wg sync.WaitGroup
 
-		// Capture goroutines
-		wg.Add(1)
-		go func() {
-			defer wg.Done()
-			data, err := pprofGoroutine(ctx, c.cachedClient)
-			if err != nil {
-				c.captureError("pprof.goroutine", err)
-				return
-			}
+		for _, target := range []string{"threadcreate", "allocs", "block", "mutex", "goroutine", "heap"} {
```

Comment:
Are the `profile`, `symbol` and `trace` intentionally omitted?

Reply:
Yes, they're handled using a different mechanism.